### PR TITLE
Viz components - Files themes refactor

### PR DESCRIPTION
**Pull request by alexanderPolosatov**

### CODE REVIEW

File: packages/devextreme/js/__internal/viz/core/themes/generic/contrast.ts [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+const BLACK = '#000000';
+const CONTRAST_ACTIVE = '#cf00da';
+const MARKER_COLOR = '#f8ca00';
+const AREA_LAYER_COLOR = '#686868';
+
+export default [{
+  theme: {
+    name: 'generic.contrast',
+    defaultPalette: 'Bright',
+    // CONTRAST_ACTIVE
```
Comment (ksercs) [inline packages/devextreme/js/__internal/viz/core/themes/generic/contrast.ts]: what does this comment mean?

File: packages/devextreme/js/__internal/viz/core/themes/generic/carmine.ts [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+          color: ACCENT_COLOR,
+        },
+        // eslint-disable-next-line spellcheck/spell-checker
+        textcloud: {
+          color: ACCENT_COLOR,
+        },
+      },
+    },
+  },
+  baseThemeName: 'generic.light',
```
Comment (ksercs) [inline packages/devextreme/js/__internal/viz/core/themes/generic/carmine.ts]: Don't u want move `baseThemeName` to the beginning everywhere? I think now it's more difficult to find it

**Review by ksercs — DISMISSED** [review]

File: packages/devextreme/js/__internal/viz/core/themes/generic/carmine.ts (hunk 2 above)
Comment (alexanderPolosatov) [reply packages/devextreme/js/__internal/viz/core/themes/generic/carmine.ts]: fixed

File: packages/devextreme/js/__internal/viz/core/themes/generic/contrast.ts (hunk 1 above)
Comment (alexanderPolosatov) [reply packages/devextreme/js/__internal/viz/core/themes/generic/contrast.ts]: fixed

**Review by EugeniyKiyashko — DISMISSED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**